src/librustc_incremental/calculate_svh.rs

```diff
@@ -69,7 +70,7 @@ impl<'a, 'tcx> SvhCalculate for TyCtxt<'a, 'tcx, 'tcx> {
         // to avoid hashing the AttrId
         for attr in &krate.attrs {
             debug!("krate attr {:?}", attr);
-            attr.node.value.hash(&mut state);
+            attr.meta().hash(&mut state);
```

Comment:
I find the name `meta` to be not useful, maybe you should rename it? (Could be a follow-up PR).